priv/rebar/boss_db_rebar.erl

```diff
@@ -99,5 +113,50 @@ compile_model(Source, Target, BossDbOpts, RebarConfig) ->
             rebar_base_compiler:ok_tuple(RebarConfig, Source, Ws);
         {error, Es, Ws} ->
             rebar_base_compiler:error_tuple(RebarConfig, Source,
-                                            Es, Ws, RecordCompilerOpts)
+                Es, Ws, RecordCompilerOpts)
+    end.
+
+%% @private
+%% @doc copies `.erl' files into the `.eunit' directory
+eunit_coverage_helper(ModelDir) ->
+    {ok, Filenames} = file:list_dir(ModelDir),
+    lists:foreach(
+        fun(Filename) ->
+            copy_model_to_eunit_dir(Filename, ModelDir)
+        end,
+        Filenames
+    ).
+
+%% @private
+copy_model_to_eunit_dir(Filename, ModelDir) ->
+    %% We only want to copy erl files. If it's something else,
+    %% don't touch it.
+    case is_erl_file(Filename) of
+        true ->
+            Source = filename:join([ModelDir, Filename]),
+            Destination = filename:join([".eunit", Filename]),
+            file:copy(Source, Destination);
+        false ->
+            ok
     end.
+
+%% @private
+%% @doc simple helper to determine whether this file is an
+%% erlang file
+is_erl_file(Filename) ->
+    filename:extension(Filename) =:= ".erl".
+
+%% @private
+%% @doc Modifications are only needed if both coverage is enabled
+%% and the models are not in the src directory.
+coverage_modifications_needed(RebarConf, BossDbOpts) ->
+    ModelDirIsNotSrc = model_dir_is_not_src(BossDbOpts),
+    coverage_enabled(RebarConf) and ModelDirIsNotSrc.
```

Comment:
Kinda weird to use a straight fun call for one and assign to a var fro the other? :)